tools/packaging/rpm/istio/istio.spec

```diff
@@ -231,14 +229,12 @@ install -d -m755 $RPM_BUILD_ROOT/%{_bindir}
 install -d -m755 $RPM_BUILD_ROOT/%{_unitdir}
 
 install -m755 %{SOURCE3} $RPM_BUILD_ROOT/%{_bindir}/istio-start.sh
-install -m755 %{SOURCE4} $RPM_BUILD_ROOT/%{_bindir}/istio-node-agent-start.sh
 
 install -m644 %{SOURCE5} $RPM_BUILD_ROOT/%{_unitdir}/istio.service
-install -m644 %{SOURCE6} $RPM_BUILD_ROOT/%{_unitdir}/istio-auth-node-agent.service
 
 binaries=(pilot-discovery pilot-agent istioctl sidecar-injector mixs mixc istio_ca galley node_agent istio-iptables istio-clean-iptables)
 pushd .
-cd ISTIO/out/linux_amd64/release
+cd ISTIO/src/istio.io/istio/out/linux_amd64
```

Comment:
This conditional and shell script in RPM makes me sad. Can you take this PR a little further and ensure this RPM spec file could pass a fedora packaging review?